cluster/gce/util.sh

```diff
@@ -2709,6 +2756,90 @@ function create-loadbalancer() {
   echo "DONE"
 }
 
+
+# detach-internal-master-ip detaches internal ip from existing master.
+#
+# Assumes:
+# * EXISTING_MASTER_NAME
+# * EXISTING_MASTER_ZONE
+# * PROJECT
+# * KUBE_MASTER_INTERNAL_IP (which will be removed)
+function detach-internal-master-ip() {
+  local aliases=$(gcloud compute instances describe "${EXISTING_MASTER_NAME}" --project "${PROJECT}" --zone "${EXISTING_MASTER_ZONE}" --flatten='networkInterfaces[0].aliasIpRanges[]' --format='value[separator=':'](networkInterfaces[0].aliasIpRanges.subnetworkRangeName,networkInterfaces[0].aliasIpRanges.ipCidrRange)' | sed 's/^://' | grep -v "${KUBE_MASTER_INTERNAL_IP}" |paste -s -d';' -)
```

Comment:
:(

Reply:
It's terrible, I know. The main issue is that 'gcloud compute instances describe' returns data in different format than gcloud compute instances network-interfaces update expects, so we need to filter out $KUBE_MASTER_INTERNAL_IP and reformat.

Maybe I can simplify by making some additional assumptions (e.g. that there is only one more alias which is true for our nodes) that would make it less generic, but readable.